src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10690,6 +10786,30 @@ definitions:
         $ref: '#/definitions/PermissionsInfo'
       definition:
         $ref: '#/definitions/Definition'
+  calculateSizeResponse:
+    type: object
+    properties:
+      id:
+        type: string
+        description: Provides the id for which POST/calculateSize has been initiated.
+      size:
+        type: string
+        description: Provides a folder size in bytes.
+      calculatedAtTime:
+        type: string
+        description: Provides the time when the calculating folder size will be completed.
```

Comment:
Do you mean this?

Reply:
We meant to provide the timestamp in response when the calculation of the folder size is completed.
We've refined the message.